webapp/platform/components/package.json

```diff
@@ -53,7 +53,7 @@
     "react": "^17.0.2",
     "react-bootstrap": "github:mattermost/react-bootstrap#d821e2b1db1059bd36112d7587fd1b0912b27626",
     "react-dom": "^17.0.2",
-    "react-intl": "^5.20.0",
+    "react-intl": "*",
```

Comment:
i am not sure if we should include this for publicly release packages 🤔

Reply:
We shouldn't be using `*` for public packages, but this package isn't public yet, so this is fine for now

Reply:
Considering the future, I have added back the correct version